src/core/python2js.c

```diff
@@ -481,10 +481,20 @@ to_js(PyObject* self,
 {
   PyObject* obj = NULL;
   int depth = -1;
-  static const char* const _keywords[] = { "depth", 0 };
-  static struct _PyArg_Parser _parser = { "O|$i:to_js", _keywords, 0 };
-  if (kwnames != NULL && !_PyArg_ParseStackAndKeywords(
-                           args, nargs, kwnames, &_parser, &obj, &depth)) {
+  PyObject* pyproxies = NULL;
+  bool create_proxies = true;
+  static const char* const _keywords[] = {
+    "", "depth", "pyproxies", "create_pyproxies", 0
+  };
+  static struct _PyArg_Parser _parser = { "O|$iOp:to_js", _keywords, 0 };
```

Comment:
I know it was there before, but could you please add a link in a comment explaining how to decode that `O|$iOp` value ?